src/main/java/com/minecolonies/tileentities/TileEntityColonyBuilding.java

```diff
@@ -260,6 +231,25 @@ public NBTTagCompound writeToNBT(@NotNull NBTTagCompound compound)
         return c != null ? c.getBuilding(getPosition()) : null;
     }
 
+
+
+    @NotNull
+    @Override
+    public NBTTagCompound writeToNBT(@NotNull NBTTagCompound compound)
+    {
+        super.writeToNBT(compound);
+        /*
+        if (colonyId == 0 && colony == null)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)